report/sections/introduction/about-giraf.tex

```diff
@@ -0,0 +1,17 @@
+\section{About GIRAF}
+GIRAF (Graphical Interface Resource for Autistic Folk) is a program developed for helping people with autism. 

```

Comment:
Lav eventuelt en list of terms i starten af rapporten hvor alle vores abbreviations kan findes.